tests/unit/kvmagent/test_vm_plugin_handlers.py

```
@@ -1280,6 +1280,112 @@ def test_detach_pci_device_from_host(self):
+      <target dev='vda' bus='virtio'/>
+    </disk>
+  </devices>
+</domain>
+"""
+        vm = vm_plugin.Vm.__new__(vm_plugin.Vm)
+        vm.get_migratable_xml = MagicMock(return_value=domain_xml)
+
+        disks, dest_xml = vm._build_domain_new_xml({})
+
+        assert disks == []
+        root = ET.fromstring(dest_xml)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)